tests/test_unit.py

```diff
@@ -61,51 +71,21 @@ def __exit__(self, *args):
 
 BUFFER_OUTPUT = True
 
-class ParseTestCase(ppt.ParseResultsAsserts, TestCase):
-    def __init__(self):
-        super().__init__(methodName='_runTest')
-
-    def _runTest(self):
-
-        buffered_stdout = StringIO()
-
-        try:
-            with resetting(sys, 'stdout', 'stderr'):
-                try:
-                    if BUFFER_OUTPUT:
-                        sys.stdout = buffered_stdout
-                        sys.stderr = buffered_stdout
-                    print(">>>> Starting test", str(self))
-                    with ppt.reset_pyparsing_context():
-                        self.runTest()
-
-                finally:
-                    print("<<<< End of test", str(self))
-                    print()
-
-        except Exception as exc:
-            if BUFFER_OUTPUT:
-                print()
-                print(buffered_stdout.getvalue())
-            raise
```

Comment:
As discussed here https://github.com/orgs/pyparsing/teams/pyparsing-dev/discussions/5?from_comment=11#discussion-5-comment-11 unittest does this.